tests/cmake/find_pkg_config/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.0)
+project(find_pkg_config)
+
+message(STATUS "PKG_CONFIG_LIBDIR: $ENV{PKG_CONFIG_LIBDIR}")
+
+find_package(PkgConfig REQUIRED)
```

Comment:
How does this file on machines that don't have pkg-config.

If might be nice to tell the person running the test exactly what is wrong (imagine a windows developer running this test for example).

Reply:
Fails and drops following into stderr:
```
CMake Error at C:/ProgramData/chocolatey/lib/cmake.portable/tools/cmake-3.23.2-windows-x86_64/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
```
Which could be confusing. Let me add a skip branch that checks whether `pkg-config` is available in the PATH.

Reply:
(otherwise the test silently skips on new installations).